src/BaGet/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -99,7 +99,7 @@ public static IServiceCollection AddBaGetContext(this IServiceCollection service
 
         public static IServiceCollection ConfigureHttpServices(this IServiceCollection services)
         {
-            services.AddMvc();
+            services.AddMvc().AddApplicationPart(typeof(BaGet.Web.Routes).Assembly);

```

Comment:
I haven't had any issues loading controllers from `BaGet.Web`. How did you run into these issues?

Also, I'm considering moving the controllers back to the `BaGet` assembly and renaming `BaGet.Web` to `BaGet.Protocol`. What do you think? The discussion for that is here: https://github.com/loic-sharma/BaGet/issues/73

Reply:
Could you add a comment like:

```
// TODO: Delete this
// See: https://github.com/loic-sharma/BaGet/issues/73
```

Reply:
comment added and added a few informations (links) to 73

Reply:
I just ran into this. Definitely needed.
@loic-sharma please merge this. It just adds a test and fixes assembly loading so no point in waiting.